lib/helpers/config-helper.js

```diff
@@ -176,10 +178,11 @@ var api = {
     try {
       var urlParts = url.parse(urlString);
       var result   = {
-        database: urlParts.path.replace(/^\//,  ''),
+        database: urlParts.pathname.replace(/^\//,  ''),
```

Comment:
likely it wont work with `sqlite:memory` url but we dont need cli / migration for in memory setups.